src/app/country/show-city/show-city.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="container">
+  <div class="capital-container">
+    <div class="select-country" *ngIf="country == undefined; else capital">
```

Comment:
``*ngIf="country; else capital"``

Reply:
I have been to try but unfortunately doesn't work because needs pipe async needs to be resolved before knowing what is in the variable. I changed for null because I changed the behavior as well